src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,3 @@
+.UserInfo {
+  font-style: italic;
```

Comment:
Consider using a different CSS property instead of `font-style` to convey the intended style. `font-style` is typically used for italicizing text for emphasis, not for styling entire components. 🎨